pkg/translator/zipkin/zipkinv2/from_translator.go

```diff
@@ -178,6 +170,20 @@ func spanToZipkinSpan(
 	return zs, nil
 }
 
+func populateZipkinSpanError(status pdata.SpanStatus, zs *zipkinmodel.SpanModel, tags map[string]string) {
+	tags[conventions.OtelStatusCode] = status.Code().String()
```

Comment:
Should this be checking for OK or ERROR only? https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk_exporters/zipkin.md#status

>Name of the code, either OK or ERROR. MUST NOT be set if the code is UNSET.

Reply:
Good point, I will address that in the PR as well (this piece of code was essentially just moved). Also the deletion of "error":false if present

Reply:
Updated the code, see my other note though - I hope I read the specs right